src/components/qas/questions/qaCalculator/index.jsx

```diff
@@ -197,13 +212,18 @@ class QACalculator extends React.PureComponent {
                     <TextField
                       id={`text-input-${id}-${defaultValue}`}
                       data-testid={`qa-calc-input-${i}`}
-                      className={fieldClasses}
+                      data-default-value={defaultValue}
+                      className={classnames(fieldClasses, {
+                        'no-label': !inputLabel,
+                      })}
                       leftIcon={leftIcon}
                       rightIcon={rightIcon}
+                      label={inputLabel}
                       placeholder={placeholder}
                       lineDirection="center"
                       type="number"
-                      min="0"
+                      min={min}
+                      max={max}
```

Comment:
so good news/bad news.  The good news is you [preempted a comment I was going to make](https://jira.lsstcorp.org/browse/EPO-3501) regarding allowing a range of values as Lauren suggested: .  Excellent.  Bad News is these min/max props are not doing anything because a number input only cares about the min and max when you're iterating the number using the lil up and down arrows, or if the input is submitted as part of a form.  We never use a form element, and there is no submit, so the min and max are never honored.  

Now, we do control the onChange handler, and we can control the value attribute, so we could make this a "controlled" input and screen values against the min and max ourselves, but I have done that before on other projects and it always gets messy.  So, instead, I suggest we leave this as-is and I'm gonna backlog the ticket I made until we have a better idea about it.

Reply:
Yeah, I had something for this inside the onBlur handler and well, it wasn't updated after the input was changed...I wanted to look into it more, but decided not to at this time. Certain worth the visit down the road. There was much more work on this than you see now that i've ripped out.